locust/main.py

```diff
@@ -392,6 +399,11 @@ def main():
         logger.error("Locust can not run distributed with the web interface disabled (do not use --no-web and --master together)")
         sys.exit(0)
 
+    if options.no_reset_stats:
+        runners.RESET_STATS_AFTER_HATCHING = False
+    if not runners.RESET_STATS_AFTER_HATCHING:
```

Comment:
This `if` condition is unnecessary in my opinion. Why not just log when setting the flag to False?